timescaledb/overview/release-notes/index.md

```diff
@@ -110,6 +113,35 @@ for more information and links to installation instructions when upgrading from
 </highlight>
 
 ## Release notes
+## 2.7.1 (2022-07-07)
+
+**Bugfixes**
+* #4494 Handle timescaledb versions aptly in multinode
+* #4493 Segfault when executing IMMUTABLE functions
+* #4482 Fix race conditions during chunk (de)compression
+* #4367 Improved buffer management in the copy operator
+* #4375 Don't ask for orderby column if default already set
+* #4400 Use our implementation of `find_em_expr_for_rel` for PG15+
+* #4408 Fix crash during insert into distributed hypertable
+* #4411 Add `shmem_request_hook`
+* #4437 Fix segfault in subscription_exec
+* #4442 Fix perms in copy/move chunk
+* #4450 Retain hypertable ownership on `attach_data_node`
+* #4451 Repair numeric partial state on the fly
+* #4463 Fix empty bytea handlng with distributed tables
```

Comment:
```suggestion
* #4463 Fix empty byte handling with distributed tables
```

Reply:
It should remain bytea, this is the name of this type:
https://www.postgresql.org/docs/current/datatype-binary.html

Reply:
bytea is a postgres datatype, so the version in PR is correct
https://www.postgresql.org/docs/current/datatype-binary.html